### ISSUE TITLE
Unable to run - stylish not installed correctly?

### ISSUE THREAD

**Issue by jamesmorgan** [ConsenSys/truffle-security#247]
After installing and running on a new truffle project I seem to always get the following error:

```
There was a problem loading formatter option: stylish
Error: Cannot find module 'eslint/lib/formatters/stylish'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at getFormatter (node_modules/truffle-security/utils/reports.js:187:16)
    at doReport (/node_modules/truffle-security/utils/reports.js:67:27)
    at process._tickCallback (internal/process/next_tick.js:68:7) code: 'MODULE_NOT_FOUND' }
Unexpected Error occured. return value of analyze should be either 0 or 1
```

**Comment by b-mueller** [comment]
Same problem with `top` also:

```
$ truffle run verify --style top 
> Warning: possible unsupported (undocumented in help) command line option: --style
Welcome to MythX! You are currently running in Free mode.

EtherBank |****************************************************************************************************| 100% || Elapsed: 2.9s ✓ completed
  Ownable |****************************************************************************************************| 100% || Elapsed: 1.5s ✓ completed
    Token |****************************************************************************************************| 100% || Elapsed: 3.9s ✓ completed
TokenSale |****************************************************************************************************| 100% || Elapsed: 4.9s ✓ completed
    Token |****************************************************************************************************| 100% || Elapsed: 3.8s ✓ completed
Error:  { Error: 
There was a problem loading formatter option: top 
Error: Cannot find module 'eslint/lib/formatters/top'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at getFormatter (/Users/bernhardmueller/.nvm/versions/node/v11.15.0/lib/node_modules/truffle-security/utils/reports.js:187:16)
    at doReport (/Users/bernhardmueller/.nvm/versions/node/v11.15.0/lib/node_modules/truffle-security/utils/reports.js:67:27)
    at processTicksAndRejections (internal/process/task_queues.js:86:5) code: 'MODULE_NOT_FOUND' }
Unexpected Error occured. return value of analyze should be either 0 or 1
```

**Comment by SDVII** [comment]
bump

**Comment by danielrea** [comment]
This should be fixed in v1.6.2 please reopen if you are still having any issues.
